.yarnrc.yml

```diff
@@ -1,5 +1,3 @@
-nodeLinker: node-modules
```

Comment:
Maybe set `nodeLinker: pnp` explicitly to ensure consistency with changes in default behaviour in newer yarn versions?